src/SprykerEco/Zed/Econda/EcondaConfig.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda;
+
+use Exception;
+use Spryker\Shared\Application\ApplicationConstants;
+use Spryker\Zed\Kernel\AbstractBundleConfig;
+use SprykerEco\Shared\Econda\EcondaConstants;
+
+class EcondaConfig extends AbstractBundleConfig
+{
+    public const ECONDA_CSV_DELIMITER = '|';
```

Comment:
duplicated variable ; if the environemnt constant is removed, this will be also a `protected + getter` thing

Reply:
done